go.mod

```diff
@@ -287,6 +287,7 @@ require (
 	github.com/pkg/browser v0.0.0-20210911075715-681adbf594b8 // indirect
 	github.com/pkg/errors v0.9.1 // indirect
 	github.com/pmezard/go-difflib v1.0.0 // indirect
+	github.com/pquerna/cachecontrol v0.1.0 // indirect
```

Comment:
How did we get a new indirect dependency without changing any other dependencies? Is go.mod out of date?

Reply:
I imagine this is because our `go-oidc` "module" does not have a `go.mod`, and hence its direct dependencies get promoted to indirect dependencies in the Teleport `go.mod` as we make use of packages within `go-oidc` that use them.

Reply:
I raised https://github.com/gravitational/go-oidc/issues/9 a few days ago - but I'm wary of doing any work on `go-oidc` until platform sec/perf determine if we are moving ahead with that library or switching back to upstream.